api/models/graphite.go

```diff
@@ -0,0 +1,22 @@
+package models
+
+type GraphiteRender struct {
```

Comment:
so api/models have a bunch of types defined, I think in 2/3 categories:
* incoming requests, e.g. sets of parameters that have to be validated
* output formats
* (internal only? not sure, is Req an example of this?)

perhaps there's a way to make this distinction a bit more clear? not sure how.